Assets/Scripts/Character.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+public class Character : MonoBehaviour
```

Comment:
Forgot to write it first, let's rename it this `CharacterController`. What I want to achieve with the naming convention is to both describe the purpose of the script and what it acts on. Also, calling it a controller helps with identifying it as a script when looking at the files in a glance.

Reply:
ofc ofc. There are only two hard things in Computer Science: cache invalidation and naming things. -- Phil Karlton